src/domain/repositories/LoggerRepository.ts

```diff
@@ -0,0 +1,6 @@
+import { FutureData } from "../../data/api-futures";
+import { Log } from "../../domain/entities/Log";
+
+export interface LoggerRepository {
+    log(log: Log, isDebug?: boolean): FutureData<void>;
```

Comment:
Booleean options are not declarative on usage. Better: `options?: { boolean?: false }`